src/Localization.tsx

```diff
@@ -45,7 +47,10 @@ export const localizedTexts = {
                     DC: Fantasy (or overalls) <br/>
         
                     <a href="https://t.me/ylikellotustiedotus" target='blank' className='tg-logo'>
```

Comment:
.tg-logo doesnt have any styles anymore, so this causes the logo to be massive with english selected
```suggestion
                    <a href="https://t.me/ylikellotustiedotus" target='blank' className='app-logo'>
```